packages/sdk/src/types/ConnectorTypes.ts

```diff
@@ -54,31 +54,25 @@ export type MediaConnectorCapabilities = {
     metadata?: boolean;
 };
 
-interface ConnectorRegistrationBase {
+export interface ConnectorUrlRegistration {
     /**
-     * Url to the connector.
+     * Url to the connector. Must be a publicly available url.
      *
-     * - If source is `url`, this must be a publicly available url.
-     *
-     * - If source is `grafx`, this must be the full url to the connector GET endpoint on GraFx Environment API.
      */
     url: string;
-
-    /**
-     * Connector source type.
-     */
-    source: ConnectorRegistrationSource;
-}
-
-export interface ConnectorUrlRegistration extends ConnectorRegistrationBase {
     source: ConnectorRegistrationSource.url;
 }
 
-export interface ConnectorGrafxRegistration extends ConnectorRegistrationBase {
+export interface ConnectorGrafxRegistration {
+    /**
+     * RemoteConnectorId of the connector with `grafx` source.
+     */
+    id: Id;
```

Comment:
Add url field and mark it as deprecated so we are not strictly breaking

Reply:
Won't this change it from a compile-time error to a runtime error?

Reply:
outdated :)